app/code/Magento/Customer/Model/Customer.php

```diff
@@ -805,7 +815,7 @@ public function isConfirmationRequired()
      */
     public function getRandomConfirmationKey()
     {
-        return md5(uniqid());
```

Comment:
Please, keep as it is.
Just ignore this rule:
```
// phpcs:ignore Magento2.Security.InsecureFunction
```

Reply:
Understood, @dmytro-ch!

**implemented**

Thank you!